apps/wing-console/console/ui/src/Console.tsx

```diff
@@ -58,7 +59,46 @@ export const Console = ({
 
   let windowTitle = title ?? "Wing Console";
 
-  const appMode = layout === LayoutType.Default ? "local" : "remote";
+  const appMode = useMemo(() => {
+    return layout === LayoutType.Default || LayoutType.Vscode
+      ? "local"
+      : "remote";
```

Comment:
This seems to be off.

Reply:
it's used to hide the API and Website Resources urls for "remote" envs (Playground and Learn)

Reply:
Maybe you mean `return layout === LayoutType.Default || layout === LayoutType.Vscode`?

Reply:
oh man... fixed!